recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
> That would be amazing. I actually have another package using Fortran - Windows builds for that would also be extremely useful.

We have some relative success with `mingwpy`  ([pyspharm](https://github.com/conda-forge/pyspharm-feedstock/blob/master/recipe/meta.yaml#L21) for example), but recently the default channel is shipping `msys2` and that should make it easier to build ad ship `Fortran` extensions on Windows. (I haven't used it yet though.)